ci-operator/step-registry/workers-rhel/ibmcloud-provision/workers-rhel-ibmcloud-provision-commands.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+  echo "Try to login..."
+  "${IBMCLOUD_CLI}" login -r ${LEASED_RESOURCE} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+function checkCli() {
+  export IBMCLOUD_CLI=ibmcloud
+  export IBMCLOUD_HOME=/output
+  echo "check IBMCLOUD_CLI: ${IBMCLOUD_CLI}..."
+  echo "Pause for debug...."
+  command -v ${IBMCLOUD_CLI}
+  ${IBMCLOUD_CLI} --version
+  ${IBMCLOUD_CLI} plugin list
+}
+
+checkCli
+region="${LEASED_RESOURCE}"
+export region
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+export PULL_SECRET_PATH=${CLUSTER_PROFILE_DIR}/pull-secret
+export SSH_PUB_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-publickey
+
+infra_id=$(oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster)
+resource_group=${infra_id}
+
+# Get existing vnet info
+VPC_CONFIG="${SHARED_DIR}/customer_vpc_subnets.yaml"
+if [[ ! -f "${VPC_CONFIG}" ]]; then
+  echo "Fail to find of VPC info file ${VPC_CONFIG}, abort." && exit 1
+fi
+echo "Reading variables from ${VPC_CONFIG}..."
+vpcName=$(yq-go r "${VPC_CONFIG}" 'platform.ibmcloud.vpcName')
+region="${LEASED_RESOURCE}"
+resource_group=$(yq-go r "${VPC_CONFIG}" 'platform.ibmcloud.resourceGroupName')
+echo "Using region: ${region}  resource_group: ${resource_group} vpc: ${vpcName}"
+
+ibmcloud_login
+
+${IBMCLOUD_CLI} target -g ${resource_group} -r ${region}
+
+workdir=`mktemp -d`
+
+# Get bastion RG info
+vnet_RG=$(yq-go r "${VPC_CONFIG}" 'platform.ibmcloud.networkResourceGroupName')
```

Comment:
I see many resource group variables are defined, 
`Line#34: resource_group=${infra_id}`
`Line#44: resource_group=$(yq-go r "${VPC_CONFIG}" 'platform.ibmcloud.resourceGroupName') `
and this one,

seen from the job log, they're supposed to be the same one, right?